### ISSUE TITLE
getHolidays() and getHolidayByDate() should return region array 

### ISSUE THREAD

**Issue by s-schmoll** [sfakir/feiertagejs#73]
It would be handy, if `getHolidays()` and `getHolidayByDate()` could return an array with regions if someone calls those functions with region `ALL`.

The type `Holiday` could be extended like so:
```
name: HolidayType
date: date
trans: func
dateString: string
equals: func
regions: array
```

It can be important to know, in with region this date is a holiday without calling `isHoliday(date, region)` for all regions.

**Comment by sfakir** [comment]
Hi s-schmoll,

I got your point, you want to receive the holiday and want to know which region it's valid in. 
Sounds logical, let me check it and come back to you.

Simon

**Comment by sfakir** [comment]
Hi @s-schmoll,

I added regions like this. It is released in latest version 1.4.0. 
Let me know if this solved your request.

Simon

```
 {
    name: 'REFORMATIONSTAG',
    date: 2020-10-30T23:00:00.000Z,
    dateString: '2020-10-31',
    regions: [
      'NI', 'BB', 'MV',
      'SN', 'ST', 'TH',
      'HB', 'HH', 'NI',
      'SH'
    ],
    translate: [Function: translate],
    getNormalizedDate: [Function: getNormalizedDate],
    equals: [Function: equals]
  },
```
